### PR TITLE
124 all subnets grouping

### PR DESCRIPTION
**Pull request by ShiriMoran**

### CODE REVIEW

**Comment by ShiriMoran** [comment]
solves #124

File: pkg/ibmvpc/examples/acl_testing5_oldsubnetsBased_withPGW.txt [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
 sub1-2-ky => sub1-1-ky : protocol: TCP
 sub1-2-ky => sub1-3-ky : protocol: TCP
 sub1-3-ky => sub1-1-ky : protocol: TCP
 sub1-3-ky => sub1-2-ky : protocol: TCP
-sub2-1-ky => Public Internet [8.8.8.8/32] : protocol: UDP dst-ports: 53
+sub2-1-ky => Public Internet 8.8.8.8/32 : protocol: UDP dst-ports: 53
 sub2-1-ky => sub2-2-ky : All Connections
 sub2-2-ky => sub2-1-ky : All Connections
+
+connections are stateful unless marked with *
```
Comment (adisos) [inline pkg/ibmvpc/examples/acl_testing5_oldsubnetsBased_withPGW.txt:13]: we do not yet have any implemented computation of whether subnets connectivity is stateful.
We should remove that comment , and have another issue to handle stateful computation for this as well,

Comment (ShiriMoran) [reply pkg/ibmvpc/examples/acl_testing5_oldsubnetsBased_withPGW.txt]: Opened https://github.com/np-guard/vpc-network-config-analyzer/issues/138 (next in my todos) as well as ugly WA code that will be removed once the issue is solved

**Review by zivnevo — APPROVED** [review]
LGTM

We have to consider if `EndpointElem` is really required in the code, given that we already have `VPCResourceIntf`. This may allow us to reduce code duplication, as `Node` also embeds `VPCResourceIntf`. This can wait for now, but maybe open an issue for that...

**Comment by ShiriMoran** [comment]
> LGTM
> 
> We have to consider if `EndpointElem` is really required in the code, given that we already have `VPCResourceIntf`. This may allow us to reduce code duplication, as `Node` also embeds `VPCResourceIntf`. This can wait for now, but maybe open an issue for that...

https://github.com/np-guard/vpc-network-config-analyzer/issues/141 
Btw - my (not yet ready) next PR further uses endpoint element

**Pull request merged**